llarp/dht/context.cpp

```diff
@@ -41,7 +41,7 @@ namespace llarp
       void
       StoreRC(const RouterContact rc) const override
```

Comment:
As per discussion, this function can probably go away entirely, as the calling point should no longer need it.